analyses/efo-mondo-mapping/README.md

```diff
@@ -0,0 +1,15 @@
+##EFO and MONDO mapping
+
+The Experimental Factor Ontology (EFO) is a resource that provides systematic description of experimental variables including cancer groups available in EBI databases and some other external projects. The scope of EFO is to support the annotation, analysis and visualization of data handled by many groups at the EBI and as the core ontology for OpenTargets.org
```

Comment:
```suggestion
The Experimental Factor Ontology (EFO) is a resource that provides systematic description of experimental variables including cancer groups available in EBI databases and some other external projects. The scope of EFO is to support the annotation, analysis and visualization of data handled by many groups at the EBI and as the core ontology for OpenTargets.org
```

Can you add what EBI is here? Also, will you add a hyperlink for OpenTargets.org and update this to be one sentence per line?

Reply:
Done. Please take another look and let me know how you like it.